android/app/src/main/java/com/threed/printmatcher/model/Submission.kt

```diff
@@ -0,0 +1,15 @@
+package com.threed.printmatcher.model
+
+import java.io.Serializable
+import java.util.*
+
+data class Submission constructor(
```

Comment:
This is a new model, it is created after the `Request` gets a commitment quantity and a commitment date. We can change / adjust this later